### PR TITLE
Feature/add navigation

### PR DESCRIPTION
**Pull request by kai-franke**

### CODE REVIEW

File: components/Navigation/NavigationBar.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+const ICON_WIDTH = "1.5rem";
+
+export default function NavigationBar() {
+  return (
+    <nav>
+      <NavigationItems>
+        <NavigationItem
+          path={"/"}
+          pageName={"Home"}
```
Comment (Roland-Hufnagel) [inline components/Navigation/NavigationBar.js:15]:
```suggestion
          path="/"
          pageName="Home"
```
you don't need to wrap strings in curly braces.

File: components/Navigation/NavigationBar.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          icon={<HomeIcon width={ICON_WIDTH} />}
```
Comment (Roland-Hufnagel) [inline components/Navigation/NavigationBar.js:16]: Sehr cool, JSX als prop zu übergeben 💪

File: components/Navigation/NavigationItem.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        {icon}
+        <p>{pageName}</p>
+      </StyledLink>
+    </StyledListElement>
+  );
+}
+
+const StyledListElement = styled.li`
+  all: unset;
+  flex: 1;
```
Comment (Roland-Hufnagel) [inline components/Navigation/NavigationItem.js:20]: bin gespannt, was das macht!

File: components/icons/DashboardIcon.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+export default function DashboardIcon({ width, fillColor }) {
+  return (
```
Comment (Roland-Hufnagel) [inline components/icons/DashboardIcon.js:2]: you don't pass ``fillColor`` as a prop.

File: pages/index.js [hunk 5]
```diff
@@ -9,6 +9,7 @@ import { Button } from "../components/Button";
 
 import { vocs } from "../lib/db";
 import Form from "../components/Form";
+import NavigationBar from "../components/Navigation/NavigationBar";
```
Comment (Roland-Hufnagel) [inline pages/index.js:12]: What for?

**Review by Roland-Hufnagel — APPROVED** [review]
Looks pretty good. Some minor important comments....

File: pages/index.js (hunk 5 above)
Comment (kai-franke) [reply pages/index.js:12]: Leftover from the development process. :)

**Pull request merged**